From 9e9bc5223c49bc51dbfd266f5dc89f2874016622 Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <i.gnatenko.brain@gmail.com>
Date: Sat, 21 Mar 2015 17:11:04 +0300
Subject: [PATCH] install: install rpm macros if RPM bin in system. Closes #56

Signed-off-by: Igor Gnatenko <i.gnatenko.brain@gmail.com>
---
 install_meson.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/install_meson.py b/install_meson.py
index 616b635..2c13046 100755
--- a/install_meson.py
+++ b/install_meson.py
@@ -107,7 +107,7 @@ if os.path.exists(module_dir):
     shutil.rmtree(module_dir)
 shutil.copytree('modules', module_dir)
 
-if os.path.exists(os.path.join('/usr', rpmmacros_dir)):
+if os.path.exists('/usr/bin/rpm'):
     print('Installing RPM macros to %s.' % rpmmacros_dir)
     outfilename = os.path.join(rpmmacros_dir, 'macros.meson')
     os.makedirs(rpmmacros_dir, exist_ok=True)
-- 
2.3.3

