From 0ba1d545afe021e09090f434d5242ae942e7d5b4 Mon Sep 17 00:00:00 2001
From: Jussi Pakkanen <jpakkane@gmail.com>
Date: Wed, 20 May 2015 22:32:55 +0300
Subject: [PATCH] Accept .S files as assembler too.

---
 compilers.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/compilers.py b/compilers.py
index 6fe7370..5e66b23 100644
--- a/compilers.py
+++ b/compilers.py
@@ -1043,7 +1043,7 @@ class GnuCCompiler(CCompiler):
         return get_gcc_soname_args(self.gcc_type, shlib_name, path, soversion)
 
     def can_compile(self, filename):
-        return super().can_compile(filename) or filename.split('.')[-1] == 's' # Gcc can do asm, too.
+        return super().can_compile(filename) or filename.split('.')[-1].lower() == 's' # Gcc can do asm, too.
 
 class GnuObjCCompiler(ObjCCompiler):
     std_warn_args = ['-Wall', '-Wpedantic', '-Winvalid-pch']
@@ -1120,7 +1120,7 @@ class ClangCCompiler(CCompiler):
         return 'pch'
 
     def can_compile(self, filename):
-        return super().can_compile(filename) or filename.split('.')[-1] == 's' # Clang can do asm, too.
+        return super().can_compile(filename) or filename.split('.')[-1].lower() == 's' # Clang can do asm, too.
 
     def get_pch_use_args(self, pch_dir, header):
         # Workaround for Clang bug http://llvm.org/bugs/show_bug.cgi?id=15136
-- 
2.4.1

