From e96a31cce4bec7e734ef56624212441e4702cf0e Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <i.gnatenko.brain@gmail.com>
Date: Fri, 30 Oct 2015 16:50:26 +0100
Subject: [PATCH] rpm macros: export *FLAGS

Signed-off-by: Igor Gnatenko <i.gnatenko.brain@gmail.com>
---
 data/macros.meson | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/data/macros.meson b/data/macros.meson
index 1da409c..8aa5555 100644
--- a/data/macros.meson
+++ b/data/macros.meson
@@ -1,11 +1,11 @@
 %__meson /usr/bin/meson
 
 %meson() %{expand:\
-  CFLAGS="%{optflags}" ; \
-  CXXFLAGS="%{optflags}" ; \
-  FFLAGS="%{optflags} -I%{_fmoddir}" ; \
-  FCFLAGS="%{optflags} -I%{_fmoddir}" ; \
-  LDFLAGS="%{__global_ldflags}" ; \
+  export CFLAGS="%{optflags}" ; \
+  export CXXFLAGS="%{optflags}" ; \
+  export FFLAGS="%{optflags} -I%{_fmoddir}" ; \
+  export FCFLAGS="%{optflags} -I%{_fmoddir}" ; \
+  export LDFLAGS="%{__global_ldflags}" ; \
   %__meson %{?1} \\\
         --prefix=%{_prefix} \\\
         --buildtype=plain \
-- 
2.6.2

