From d4257000014ade5d8dc2e7baf281ceababf11a36 Mon Sep 17 00:00:00 2001
From: Ninja-Koala <mail@ninjakoa.la>
Date: Sun, 17 Mar 2019 15:31:14 +0100
Subject: [PATCH 1/2] Fix #5046

---
 mesonbuild/modules/gnome.py | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/mesonbuild/modules/gnome.py b/mesonbuild/modules/gnome.py
index 4473bcbf..8833a214 100644
--- a/mesonbuild/modules/gnome.py
+++ b/mesonbuild/modules/gnome.py
@@ -189,9 +189,13 @@ class GnomeModule(ExtensionModule):
                 gresource_ld_binary = True
 
         gresource = kwargs.pop('gresource_bundle', False)
-        if gresource or gresource_ld_binary:
+
+        if gresource:
             g_output = args[0] + '.gresource'
             g_name = args[0] + '_gresource'
+        elif gresource_ld_binary:
+            g_output = args[0] + '_ld_binary.gresource'
+            g_name = args[0] + '_ld_binary_gresource'
 
         output = args[0] + '.c'
         name = args[0] + '_c'
@@ -240,10 +244,7 @@ class GnomeModule(ExtensionModule):
         if gresource or gresource_ld_binary:
             target_g = GResourceTarget(g_name, state.subdir, state.subproject, g_kwargs)
             if gresource: # Only one target for .gresource files
-                if target_g.get_id() not in self.interpreter.build.targets:
-                    return ModuleReturnValue(target_g, [target_g])
-                else:
-                    return ModuleReturnValue(target_g, [])
+                return ModuleReturnValue(target_g, [target_g])
 
         target_c = GResourceTarget(name, state.subdir, state.subproject, kwargs)
 
-- 
2.21.0

