From d6b119dd63869631ccb4c72e002406a65a5060d5 Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <i.gnatenko.brain@gmail.com>
Date: Tue, 24 Jul 2018 15:55:14 +0200
Subject: [PATCH 4/4] rpm: set --auto-features=auto

Signed-off-by: Igor Gnatenko <i.gnatenko.brain@gmail.com>
---
 data/macros.meson | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/macros.meson b/data/macros.meson
index 4a8a3ca4..39ceca23 100644
--- a/data/macros.meson
+++ b/data/macros.meson
@@ -1,6 +1,6 @@
 %__meson %{_bindir}/meson
 %__meson_wrap_mode nodownload
-%__meson_auto_features enabled
+%__meson_auto_features auto
 
 %meson \
     %set_build_flags \
-- 
2.18.0

